helix-stdx/src/faccess.rs

```diff
@@ -29,6 +29,15 @@ mod imp {
     use std::os::unix::fs::{MetadataExt, PermissionsExt};
 
     pub fn access(p: &Path, mode: AccessMode) -> io::Result<()> {
+        #[cfg(target_os = "linux")]
```

Comment:
gated behind `linux` as `CAP_DAC_OVERRIDE` does not exist on other unix-like kernels. See also https://docs.rs/rustix/1.1/rustix/thread/struct.CapabilitySet.html